src/globus_cli/commands/timer/create.py

```diff
@@ -0,0 +1,288 @@
+import datetime
+from typing import List, Optional, Tuple, cast
+
+import click
+from globus_sdk import TimerJob, TransferData
+
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import command, group, mutex_option_group
+from globus_cli.termio import FORMAT_TEXT_RECORD, formatted_print
+
+from ._common import (
+    DATETIME_FORMATS,
+    JOB_FORMAT_FIELDS,
+    START_HELP,
+    parse_timedelta,
+    read_csv,
+    require_activated_endpoints,
+)
+
+INTERVAL_HELP = """
+    Interval at which the job should run. Use 'w', 'd', 'h', 'm', and 's' as suffixes
+    to specify weeks, days, hours, minutes, and seconds. Examples: '1h 30m', '500s',
+    '24h', '1d 12h', '2w', etc. Must be in order: hours -> minutes -> seconds. You
+    should either use quotes ('1d 2h') or write without spaces (1d2h).
+"""
+
+
+@group("create", short_help="Submit a Timer job", hidden=True)
+def create_command():
+    pass
+
+
+@command("transfer", short_help="Create a recurring transfer job in Timer")
+@click.option(
+    "--name",
+    required=True,
+    type=str,
+    help="Name to identify this job in the timer service (not necessarily unique)",
+)
+@click.option(
+    "--start",
+    required=False,
+    type=click.DateTime(formats=DATETIME_FORMATS),
+    help=START_HELP,
+)
+@click.option(
+    "--interval",
+    required=False,
+    type=str,
```

Comment:
I'm not sure I want to pursue this right this minute, but it seems like instead of a `str`, we could do the parsing at `click` parse-time, and have a custom type here to do the work.

Reply:
Yeah, it would be nice, though likewise I think maybe that's a good target for a future refactor.